### DISCUSSION TITLE
Bias in reranking algorithm

### DISCUSSION THREAD

**Discussion by xinyuluo** [osm-search/Nominatim#2626] [Q&A] [answered]
Will Nominatim consider adding bias to reranking algorithm that considers proximity to user location? Thanks.

**Comment by lonvia** [comment, marked as answer]
There is the `viewbox` parameter that does something like that. There are no plans to add any other proximity features.

**Reply by xinyuluo** [reply]
@lonvia please correct me if my understanding is wrong, i believe `viewbox` only affects choosing the pool of results limited by the `limit` parameter. and after that, the reranking algorithm is based on importance scores and how well each place addresses the query, and doesn't take into account `viewbox`. Is that right?

**Reply by lonvia** [reply]
`viewbox` only works as a filter when `bounded=1`. Otherwise (and per default) it works as a reranking parameter giving preference to objects in the given viewbox area.
